htdocs/projet/class/project.class.php

```diff
@@ -499,6 +500,7 @@ public function fetch($id, $ref = '')
         if (!empty($id))
         {
         	$sql .= " WHERE rowid=".$id;
+			$sql .= " AND entity IN (".getEntity('project').")";
```

Comment:
@mapiolca 
a rowid is unique and it is necessarily linked to a single entity.
What is the goal ?

Reply:
If you loop all project and dev don't filter entities,  others's entities projects will be shown.

Reply:
@mapiolca  on which page of Dolibarr?

Reply:
It's an issue seen in Doc2Project Module

Reply:
@mapiolca maybe a missing entity filter in this module !

Reply:
@hregis Yes it is, I submit a pull request to ATM, but I thought that would be better to add a security to prevent future issues

Reply:
If you think it's useless, I delete

Reply:
@mapiolca yes sorry, it's not necessary, rowid is unique and fetch function return only one record

Reply:
No filter on entity when fetching on rowid.